assets/js/pages/SapSystemsOverviewPage/SapSystemsOverview.test.jsx

```diff
@@ -80,13 +90,69 @@ describe('SapSystemsOverviews component', () => {
         tenant
       );
       expect(mainRow.querySelector('td:nth-child(5)')).toHaveTextContent(
-        dbAddress
+        sapSystemType
       );
       expect(mainRow.querySelector('td:nth-child(6)')).toHaveTextContent(
+        dbAddress
+      );
+      expect(mainRow.querySelector('td:nth-child(7)')).toHaveTextContent(
         'ENSA1'
       );
     });
 
+    it('should display the correct SAP system type', () => {
+      const sapSystemTypes = [
+        'ABAP',
+        'J2EE',
+        'ABAP|J2EE',
```

Comment:
I understand what you are trying to test, but this entry you are putting is not a real scenario.
Here, if you want to test ABAP and JAVA together, you should have one instance with `ABAP` features and other instance with `J2EE` features. Not 1 instance with both, which doesn't exist